test/LD.c

```diff
@@ -26,8 +32,8 @@ int main(int argc, char **argv) {
     
     //Set up circuit parameters
     int q, r;
```

Comment:
What are `q` and `r`? Could you add a comment explaining?

Reply:
I added more comments